docker/observation/README.md

```
@@ -0,0 +1,146 @@
+Username: admin
+Password: admin
+```
+
+Pre-configured datasources:
+- **Elasticsearch** (default): Log search and analysis
+- **Prometheus**: Metrics queries
+- **Tempo**: Trace exploration
+
+### Direct Access
+
+- **Prometheus**: http://localhost:9090
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)